static/app/views/preprod/buildComparison/main/sizeCompareSelectedBuilds.tsx

```diff
@@ -70,6 +70,25 @@ function BuildButton({buildDetails, icon, label, onRemove}: BuildButtonProps) {
   );
 }
 
+const ComparisonContainer = styled(Flex)`
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  gap: ${p => p.theme.space.lg};
```

Comment:
Not sure with styled components but you might be able to get this with the `gap="lg"` prop. Same as width/justify/align.